devDocs/startupShutdown.md

```diff
@@ -0,0 +1,179 @@
+# NVDA Starting and Exiting outline
+
+## Ways to start NVDA:
+
+1. For an installed copy:
+    1. Ctrl+Alt+N (Desktop shortcut)
+        - test: `startupShutdownNVDA.Starts from desktop shortcut`
+    1. Automatically via Ease of Access on the Windows sign-in screen (at boot or signing out of a previous session)
+    1. Automatically via Ease of Access on User Account Control (UAC) screens
+    1. Automatically by Ease of Access after signing in to Windows
+1. For an installed copy, portable copy, installer:
+    1. An exiting instance of NVDA starting a new process (see shutting down procedures)
+    1. By running the exe (see cli options too).

```

Comment:
What "cli options"? This probably should link to the apropriate section of the user guide.

Reply:
I think we determined elsewhere that "cli options" could be removed.

Reply:
```suggestion
    1. By running the exe
```